test/test_retriever.py

```diff
@@ -358,28 +371,26 @@ def test_add_dialect():
     result['dummy_key'] = 'dummy_value'
 
     add_dialect(table_dict, table)
-    assert(table_dict == result)
+    assert table_dict == result
 
 
 def test_add_schema():
     """Test adding values from schema key to python script"""
-    table = {}
-    table['schema'] = {}
     table_dict = {}
+    table = {'schema': {}}  # dictionary literal for dictionary creation
```

Comment:
Is there a reason not to just write these entirely as a dictionary literal instead of using the appends and assignments. E.g., for the `result` dictionary below I think this is clearer:

```
result = {'columns': [('col1', ('int',)), ('col2', ('char', 20))],
          'ct_column': "'ct_column'",
          'ct_names': ['ct1', 'ct2', 'ct3', 'ct4'],
          'dummy_key': 'dummy_value'
}
```

I know you didn't write these originally, but if we're going to change part of this here we might as well make the full change (or not make any changes here and make the full change elsewhere.